index.js

```diff
@@ -142,7 +247,24 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
+  if(movies.length === 0){
+    return null;
+  }
+  let moviename = '';
+  let biggestBoxOffice = (movies[0].boxOffice).substring(1);
+  biggestBoxOffice = parseInt(biggestBoxOffice) * 1000000;
+  for(let i = 1; i < movies.length; i++){
+    let boxoffices = (movies[i].boxOffice).substring(1);
+    boxoffices = parseInt(boxoffices) * 1000000;
+    if(boxoffices > biggestBoxOffice){
+      moviename = movies[i].title;
+    }
+  }
+  return moviename;
+}
```

Comment:
So close here!  What happens in the event that the _first movie_ is actually the largest box office?  What would we return and why?